### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by Regular-Baf**
it is baftacular now

Signed-off-by: Regular Baf <95348524+Regular-Baf@users.noreply.github.com>

### CODE REVIEW

**Comment by Regular-Baf** [comment]
I have no idea why this bot is complaining

**Comment by Su386yt** [comment]
Spelling errors

**Comment by Su386yt** [comment]
I'll try and combine this with #19

**Comment by Su386yt** [comment]
[Check the comment on #19 for the merged version] (https://github.com/PrismLauncher/prismlauncher.github.io/pull/19#issuecomment-1284459914)

**Comment by Regular-Baf** [comment]
I think the in depth explanations are best left for the blog. And we certainly shouldn't be naming names to avoid conflicts and the like. There is no real changes in Prism in terms or direction or philosophy, it's just PolyMC back on track with a new name.

File: src/wiki/overview/faq.md [hunk 1]
```diff
@@ -9,16 +9,50 @@ eleventyNavigation:
...
+
+To find out how, [please see here](/wiki/getting-started/migrating-multimc).
+
+## Can I keep using PolyMC? Is it safe?
+
+If modified, PolyMC versions **1.4.2 and below are probably safe to continue using** for now. With that said, **please do not update PolyMC**, as the security of future versions cannot be guaranteed.
+
+Prism Launcher is brought to you by the same trusted team beind PolyMC, and is available now across many platforms. We do encourage that you make the switch as soon Prism becomes available to you.
+
+### Users who wish to keep using PolyMC should change a few settings to ensure security...
```
Comment (DioEgizio) [inline src/wiki/overview/faq.md:48]: this part can be removed

**Comment by timoreo22** [comment]
Superseded by #15

**Pull request closed without merging**